src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.baseclasses;
+
+// using abstract class
+public abstract class Machine {
+    // abstract methods
```

Comment:
maybe not a mistake, but these comments are describing very basic things, so I would not push them